src/igvfd/schemas/publication.json

```diff
@@ -112,5 +112,12 @@
             }
         }
     },
+    "fuzzy_searchable_fields": [
+        "@type",
+        "abstract",
```

Comment:
by adding abstract you are going to get all sort of things showing up because it will bloat the search a lot.  Imagine a paragraph broken down the way that this uuid is below.  I don't think you are going to get the results you want.

8
81
81a
81a6
81a6c
81a6cc
81a6cc1
81a6cc12
81a6cc12-
81a6cc12-2
81a6cc12-28
81a6cc12-284
81a6cc12-2847
81a6cc12-2847-
81a6cc12-2847-4
81a6cc12-2847-4e
81a6cc12-2847-4e2
81a6cc12-2847-4e2e
81a6cc12-2847-4e2e-
81a6cc12-2847-4e2e-8
81a6cc12-2847-4e2e-8f
81a6cc12-2847-4e2e-8f2
81a6cc12-2847-4e2e-8f2c
81a6cc12-2847-4e2e-8f2c-
81a6cc12-2847-4e2e-8f2c-f
81a6cc12-2847-4e2e-8f2c-f5
81a6cc12-2847-4e2e-8f2c-f56
81a6cc12-2847-4e2e-8f2c-f566
81a6cc12-2847-4e2e-8f2c-f5666
81a6cc12-2847-4e2e-8f2c-f56669
81a6cc12-2847-4e2e-8f2c-f566699
81a6cc12-2847-4e2e-8f2c-f566699e
81a6cc12-2847-4e2e-8f2c-f566699eb
2
28
284
2847
4
4e
4e2
4e2e
8
8f
8f2
8f2c
f
f5
f56
f566
f5666
f56669
f566699
f566699e
f566699eb
f566699eb2
f566699eb29
8
81
81a
81a6
81a6c
81a6cc
81a6cc1
81a6cc12
8
81
81a
81a6
81a6c
81a6cc
81a6cc1
81a6cc12
81a6cc122
81a6cc1228
81a6cc12284
81a6cc122847
81a6cc1228474
81a6cc1228474e
81a6cc1228474e2
81a6cc1228474e2e
81a6cc1228474e2e8
81a6cc1228474e2e8f
81a6cc1228474e2e8f2
81a6cc1228474e2e8f2c
81a6cc1228474e2e8f2cf
81a6cc1228474e2e8f2cf5
81a6cc1228474e2e8f2cf56
81a6cc1228474e2e8f2cf566
81a6cc1228474e2e8f2cf5666
81a6cc1228474e2e8f2cf56669
81a6cc1228474e2e8f2cf566699
81a6cc1228474e2e8f2cf566699e
81a6cc1228474e2e8f2cf566699eb
81a6cc1228474e2e8f2cf566699eb2
81a6cc1228474e2e8f2cf566699eb29